### ISSUE TITLE
Support for Apple M1 chips

### ISSUE THREAD

**Issue by fabiolnm** [commitdev/zero#419]
Following [these instructions](https://getzero.dev/docs/zero/getting-started/installation):

```
% brew tap commitdev/zero 
fabio@Fabios-MacBook-Pro zero % brew install zero 
Error: formulae require at least a URL
/opt/homebrew/Library/Homebrew/formula.rb:274:in `determine_active_spec'
/opt/homebrew/Library/Homebrew/formula.rb:209:in `initialize'
/opt/homebrew/Library/Homebrew/formulary.rb:181:in `new'
/opt/homebrew/Library/Homebrew/formulary.rb:181:in `get_formula'
```

Trying to use Rosetta:

```
% arch -x86_64 brew install zero 
Updating Homebrew...
==> Auto-updated Homebrew!
Updated 1 tap (homebrew/core).
==> Updated Formulae
Updated 1 formula.

Error: Cannot install under Rosetta 2 in ARM default prefix (/opt/homebrew)!
To rerun under ARM use:
    arch -arm64 brew install ...
To install under x86_64, install Homebrew into /usr/local.
```

Any advice about how can I try to build a M1 compatible build and create a PR to add support for the Apple chips?

**Comment by bmonkman** [comment]
Hey, thanks a lot for submitting the issue! A while back I disabled cross-compiling for arm64 because I was having trouble with it during the release process but I just tried it again and it was working for me so I re-enabled it and released version 0.2.2

**Comment by fabiolnm** [comment]
Thank you very much, it worked!
